### PR TITLE
Bunch of fixes related to PEF

### PR DESCRIPTION
**Pull request by grena**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | 2 failures
| Specs & Static     | Y
| Changelog updated  | Not necessary

### CODE REVIEW

File: src/Pim/Bundle/ReferenceDataBundle/Resources/config/datagrid/query_builders.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ services:
             - '@pim_reference_data.id_resolver'
             - ['IN', 'EMPTY']
         tags:
-            - { name: 'pim_catalog.doctrine.query.filter', priority: 30 }
+            - { name: 'pim_catalog.doctrine.query.filter', priority: 15 }
```
Comment (skeleton) [inline src/Pim/Bundle/ReferenceDataBundle/Resources/config/datagrid/query_builders.yml]: why ?

Comment (grena) [inline src/Pim/Bundle/ReferenceDataBundle/Resources/config/datagrid/query_builders.yml]: We have to check if it's an attribute before checking it's a ref data

**Comment by jmleroux** [comment]
GTM

**Comment by sumbobyboys** [comment]
GTM

**Comment by skeleton** [comment]
GTM

**Pull request merged**